app/brave_settings_strings.grdp

```diff
@@ -1391,6 +1391,22 @@
     Disable mute function in tab speaker icons
   </message>
 
+  <!-- Settings / Leo Assistant -->
+  <message name="IDS_SETTINGS_LEO_ASSISTANT" desc="Title for an item in the 'Leo Assistant' section of Brave Settings">
+    Leo
+  </message>
+  <message name="IDS_SETTINGS_LEO_ASSISTANT_SHOW_ICON_ON_TOOLBAR_LABEL" desc="The text for settings option">
+    Show Leo icon in the toolbar
```

Comment:
Changed `Show Leo icon in the toolbar` -> `Show Leo icon in the sidebar`
https://github.com/brave/brave-core/pull/19212/commits/414fb6cd24fb8e0b6dfbf0eefabbcb9392065272#diff-7d70e7728c1c75eb4d1e56385b978d6fd66b1994881e7bc5f335d7e1ed519390R1399
because the issue: https://github.com/brave/brave-browser/issues/30590 says about sidebar